### PR TITLE
Pause doc: misc, fixups

### PR DESCRIPTION
**Pull request by Dragorn421**
Heterogeneous assortment of small things or things I missed before

### CODE REVIEW

**Review by fig02 — APPROVED** [review]

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_map.c [hunk 1]
```diff
@@ -764,25 +765,25 @@ void KaleidoScope_DrawWorldMap(PlayState* play, GraphicsContext* gfxCtx) {
 
         // Quad PAGE_BG_QUADS + WORLD_MAP_QUAD_28
 
-        pauseCtx->mapPageVtx[172].v.ob[0] = pauseCtx->mapPageVtx[174].v.ob[0] =
+        pauseCtx->mapPageVtx[43 * 4 + 0].v.ob[0] = pauseCtx->mapPageVtx[43 * 4 + 2].v.ob[0] =
```
Comment (cadmic) [inline src/overlays/misc/ovl_kaleido_scope/z_kaleido_map.c:768]: use `PAGE_BG_QUADS + WORLD_MAP_QUAD_28` instead of `43` here?

**Review by cadmic — APPROVED** [review]
(comment above still stands)

File: src/overlays/misc/ovl_kaleido_scope/z_kaleido_map.c (hunk 1 above)
Comment (Dragorn421) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_map.c:768]: I didn't because of how much clutter it would add, you think it's fine?

Comment (cadmic) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_map.c:768]: it's kinda weird that everything else is technically shiftable if you add a new enum (I hope), but not this one. Does it really look so bad? Could also give a name to `PAGE_BG_QUADS + WORLD_MAP_QUAD_28` instead

Comment (fig02) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_map.c:768]: would be fine with either fwiw, as long as there is a good name for a wrapper macro

Comment (Dragorn421) [reply src/overlays/misc/ovl_kaleido_scope/z_kaleido_map.c]: 43 -> `PAGE_BG_QUADS + WORLD_MAP_QUAD_28` 9c4bcb93ec39af30cda1d638d25be2d136f98d5c

**Review by fig02 — APPROVED** [review]
since these are cleanups, is this wrapping up the pause docs? or is there still more from the old PR?

**Comment by Dragorn421** [comment]
There's more still :)

**Pull request merged**